_posts/0000-01-02-anshulsinghgehlot.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
"My life has consisted in learning and forgetting and learning and forgetting and learning and forgetting statistical mechanics"